documentation/book/proc-deploying-the-topic-operator-standalone.adoc

```diff
@@ -0,0 +1,63 @@
+
+[id='deploying-the-topic-operator-standalone']
+= Deploying the Topic Operator standalone
+
+Deploying the Topic Operator as a standalone component is more complicated than installing it using the Cluster Operator, but is more flexible.
+For instance is can operate _with_ any Kafka cluster, not necessarily one deployed by the Cluster Operator.
+
+.Prerequisites
+
+* An existing Kafka cluster for the Topic Operator to connect to.
+
+.Procedure
+
+. Edit the `examples/install/cluster-operator/05-Deployment-strimzi-topic-operator.yaml` resource. You will need to change the following
++
+* The `STRIMZI_KAFKA_BOOTSTRAP_SERVERS` environment variable in `Deployment.spec.template.spec.containers[0].env` should be set to a list of bootstrap brokers in your Kafka cluster, given as a comma-separated list of `_<hostname>_:‍_<port>_` pairs.
+* The `STRIMZI_ZOOKEEPER_CONNECT` environment variable in `Deployment.spec.template.spec.containers[0].env` should be set to a list of the Zookeeper nodes, given as a comma-separated list of `_<hostname>_:‍_<port>_` pairs. This should be the same Zookeeper cluster that your Kafka cluster is using.
+* The `STRIMZI_NAMESPACE` environment variable in `Deployment.spec.template.spec.containers[0].env` should be set to the {ProductPlatformName} namespace in which you want the operator to watch for  `KafkaTopic` resources.
+
+. Deploy the Cluster Operator.
++
+ifdef::Kubernetes[]
+On {KubernetesName} this can be done using `kubectl apply`:
++
+[source,shell]
+kubectl apply -f examples/install/cluster-operator
++
+On {OpenShiftName} this can be done using `oc apply`:
+endif::Kubernetes[]
+ifndef::Kubernetes[]
+This can be done using `oc apply`:
+endif::Kubernetes[]
++
+[source,shell]
+----
+oc apply -f examples/install/cluster-operator
+----
+
+. Verify that the Topic Operator has been deployed successfully. 
++
+ifdef::Kubernetes[]
+On {KubernetesName} this can be done using `kubectl describe`:
++
+[source,shell]
+kubectl describe deployment strimzi-topic-operator
++
+On {OpenShiftName} this can be done using `oc applydescribe:
```

Comment:
typo ... `applydescribe`. And again ... this sentence is in the kubernetes section and says the same as line 50.